### ISSUE TITLE
[3.3.5a] [Object] Scarab Coffer(s)

### ISSUE THREAD

**Issue by Loginisalreadytaken** [TrinityCore/TrinityCore#9638]
http://www.wowhead.com/object=180691 (Scarab Coffer; old. for loot, www. for locations)
http://www.wowhead.com/object=180690 (Large Scarab Coffer; old. for loot, www. for locations)

Version: 3.3.5a (12340)
Core revision: Trinity Core 81908c7d7bc7+
Database revision: TDB 335.51

All coffers are missing and the items they contain seem to be slightly off. Based on the information from the old. addresses, I'd say the following:
- A Scarab Coffer should always contain a Scarab Bag (21156) and all the AQ20-idols should be grouped, 100% chance that 1 idol type drops with the quantity 1-2.
- A Large Scarab Coffer should always contain a Scarab Bag (21156) and all the AQ40-idols should be grouped, 100% chance that 2 idol types drop with the quantity 1-3.
##

**Comment by Aokromes** [comment]
This is still needed? Because Malcrom respawned AQ few months ago.

**Comment by Elarose** [comment]
Still needed, along with Ossirian crystals.
Aq20:
Scarab coffers:
These are the locations  on retail as of now seem to be static not pooled. (Five runs and all in the same position) https://postimg.org/image/xtn5eurzp/     10 Total (The tom tom coordinates from wowhead are correct)
Loots: https://postimg.org/image/q7mvikec1/

Missing. Ossirian crystals.
http://www.wowhead.com/object=180619/ossirian-crystal#comments
The only one up at the start is the one located directly infront of him, it's also the one that animates after he is killed (In the video) The others spawn when he is pulled and stay after hes died.
https://youtu.be/FyVfyiShvEc

Aq40:
Coming soon

**Comment by offl** [comment]
Yep, they shouldn't be pooled.

**Comment by Elarose** [comment]
Aq40 scarab coffer locations sniff.
`
DELETE FROM `gameobject` WHERE `guid` BETWEEN @OGUID+0 AND @OGUID+10;
INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `VerifiedBuild`) VALUES
(@OGUID+0, 180690, 531, 3428, 3428, 1, 1, -8561.12, 1989.54, -3.091515, 4.188792, 0, 0, -0.866025, 0.5000008, 7200, 255, 1, 28153),
(@OGUID+1, 180690, 531, 3428, 3428, 1, 1, -8134.142, 1125.443, -86.26643, 2.513274, 0, 0, 0.9510565, 0.3090171, 7200, 255, 1, 28153),
(@OGUID+2, 180690, 531, 3428, 3428, 1, 1, -8583.558, 1287.909, -68.88995, 1.605702, 0, 0, 0.7193394, 0.6946588, 7200, 255, 1, 28153),
(@OGUID+3, 180690, 531, 3428, 3428, 1, 1, -8674.741, 1594.589, -86.13927, 3.57793, 0, 0, -0.9762955, 0.2164421, 7200, 255, 1, 28153),
(@OGUID+4, 180690, 531, 3428, 3428, 1, 1, -8882.011, 1347.217, -92.58246, 1.134463, 0, 0, 0.5372992, 0.8433917, 7200, 255, 1, 28153),
(@OGUID+5, 180690, 531, 3428, 3428, 1, 1, -8928.15, 1823.571, -18.932, 1.099556, 0, 0, 0.5224981, 0.8526405, 7200, 255, 1, 28153), 
(@OGUID+6, 180690, 531, 3428, 3428, 1, 1, -9106.087, 1502.935, -100.0013, 5.078908, 0, 0, -0.5664063, 0.8241262, 7200, 255, 1, 28153), 
(@OGUID+7, 180690, 531, 3428, 3428, 1, 1, -9161.575, 1579.325, -79.93993, 1.884953, 0, 0, 0.8090162, 0.5877863, 7200, 255, 1, 28153), 
(@OGUID+8, 180690, 531, 3428, 3428, 1, 1, -9201.085, 1486.463, -95.17252, 2.007128, 0, 0, 0.8433914, 0.5372996, 7200, 255, 1, 28153), 
(@OGUID+9, 180690, 531, 3428, 3428, 1, 1, -9242.85, 1567.696, -77.35221, 0.4886912, 0, 0, 0.2419214, 0.9702958, 7200, 255, 1, 28153), 
(@OGUID+10, 180690, 531, 3428, 3428, 1, 1, -9203.896, 1961.314, -50.16774, 1.553341, 0, 0, 0.7009087, 0.7132511, 7200, 255, 1, 28153); 
`

**Comment by Faq** [comment]
@Killyana pls check provided info.

**Comment by offl** [comment]
Scarab bag seems to be fine and requires no changes. Same with count of bags, seems like it was changed in Cata to 5. I only removed quest requirement because they can drop without quest and changed max count of idols to 3 from 1. Now 1-3 different idols can drop. Currently it's hard or impossible to make them drop always (always at least one idol of one type) but it is also unclear if they should drop always or not.
Thanks everyone for research
